application/classes/user.py

```diff
@@ -13,52 +14,50 @@ class User:
     '''
         A class to create user objects.
     '''
-    def __init__(self, name, code, dob, role_id, country_id, username, password):
-        self.__name = name
-        self.__code = code
-        self.__dob = dob
-        self.__role_id = role_id
-        self.__country_id = country_id
-        self.__username = username
-        self.__password = password
+    def __init__(self):
+        self.__name = self.__name
```

Comment:
at the moment these lines dont do anything and will probably have a red squiggly line under them. It's worth removing these

Reply:
ah ok, sorry I thought stylistically these stay in. I will remove them.